lib/sparkql/parser_compatibility.rb

```diff
@@ -5,60 +5,49 @@ module Sparkql::ParserCompatibility
   MAXIMUM_EXPRESSIONS = 75
   MAXIMUM_LEVEL_DEPTH = 2
 
-  # TODO I Really don't think this is required anymore
   # Ordered by precedence.
```

Comment:
I left what is still required. We can spin up a new issue if we want to remove the rest.